openjdk8/src/main/docker/setup-env.bash

```diff
@@ -1,18 +1,27 @@
 #!/bin/bash
+
+isTrue() {  
+  if [[ ${1,,*} = "true" ]] ;then 
+    return ${true}
+  else 
+    return ${false}
+  fi 
+}
+
 ALPN_BOOT=
-if [[ -n "$ALPN_ENABLE" ]]; then
+if isTrue "${ALPN_ENABLE:=false}" ; then
   ALPN_BOOT="$( /opt/alpn/format-env-appengine-vm.sh )"
 fi
 
 DBG_AGENT=
-if [[ "$GAE_PARTITION" = "dev" ]]; then
-  if [[ -n "$DBG_ENABLE" ]]; then
+if isTrue "${DBG_ENABLE:=$( if [[ -z ${CDBG_DISABLE} ]] ; then echo true; else echo false ; fi )}" ; then
```

Comment:
I'm 50:50 on this.  I can see that it is a good backwards compatibility fallback... but then I can't find any documentation about it either... other than the code in the cdbg script.   Although I can imagine somebody looking at that script and then being driven crazy setting CDBG_DISABLE and wondering why it is not disabled.

Note that even if we don't respect it, we still have to unset it in our script, so we can't totally remove handling for CDBG_DISABLE.  If we have to code for it, then why not respect it?    Hmmm think I'm talking myself to 60:40 keeping it.

But its OK either way.  your call!